ldai/client.py

```diff
@@ -16,17 +16,69 @@ class LDMessage:
     content: str
 
 
-@dataclass
-class AIConfigData:
-    model: Optional[dict]
-    prompt: Optional[List[LDMessage]]
+class ModelConfig:
+    """
+    Configuration related to the model.
+    """
+
+    def __init__(self, id: str, temperature: Optional[float] = None,
+                 max_tokens: Optional[int] = None, attributes: dict = {}):
+        """
+        :param id: The ID of the model.
+        :param temperature: Turning parameter for randomness versus determinism. Exact effect will be determined by the model.
+        :param max_tokens: The maximum number of tokens.
+        :param attributes: Additional model-specific attributes.
+        """
+        self._id = id
+        self._temperature = temperature
+        self._max_tokens = max_tokens
+        self._attributes = attributes
+
+    @property
+    def id(self):
+        """
+        The ID of the model.
+        """
+        return self._id
+
+    @property
+    def temperature(self):
```

Comment:
Do these comments need to indicate optionality?

Reply:
I added return types to the method signatures. I think that should be sufficient.